openassessment/xblock/ui_mixins/mfe/mixin.py

```diff
@@ -86,13 +89,51 @@ def _submission_create(self, data):
 
     @XBlock.json_handler
     def submission(self, data, suffix=""):
-        if suffix == 'draft':
+        if suffix == HandlerSuffixes.SUBMISSION_DRAFT:
             return self._submission_draft(data)
-        elif suffix == "create":
+        elif suffix == HandlerSuffixes.SUBMISSION_SUBMIT:
             return self._submission_create(data)
```

Comment:
I like these more but they technically differ from our contract. If we want to change, need to just communicate to frontend and update in contract.

Current paths:
- POST {xblock_id}/handler/submission/submit
- POST {xblock_id}/handler/submission/draft

Expected paths:
- POST {xblock_id}/handler/submit
- POST {xblock_id}/handler/submission/draft